azurelinuxagent/common/osutil/alpine.py

```diff
@@ -32,7 +32,7 @@ def is_dhcp_enabled(self):
         return True
 
     def get_dhcp_pid(self):
-        return shellutil.run_command(["pidof", "dhcpcd"]).strip()
+        return self._get_dhcp_pid(["pidof", "dhcpcd"])
```

Comment:
Processing of the text returned by run_command is now done in a static member (_get_dhcp_pid) in the default osutil